ResourceStatusSystem/Client/SiteStatus.py

```diff
@@ -43,7 +43,6 @@ def __init__( self ):
     self.__opHelper = Operations()
     self.rssFlag = ResourceStatus().rssFlag
     self.rsClient = ResourceStatusClient()
-    self.wmsAdministrator = RPCClient( 'WorkloadManagement/WMSAdministrator' )
```

Comment:
No, it's not a bug. In services and agents, for example, the initialization of the objects is done inside, e.g., the `initialize()` method and not inside the `__init__()`: because prior to `initilize`, the CS is not fully loaded. So this is just to avoid possible issues like that.